### DISCUSSION TITLE
HTTP endpoint documentation

### DISCUSSION THREAD

**Discussion by NickPronto** [gopro/OpenGoPro#250] [Q&A]
I'm building a system where the cameras will be fairly hard to reach. Is there a way to use the HTTP get requests to set an endpoint to a locally stored NAS? 

These cameras won't have cell connection or the ability to connect to the cloud service, which is why I'm hoping to collate a few cameras onto one NAS for removal and upload to the cloud off site?

**Comment by KonradIT** [comment]
Does the NAS have ability to run scripts and interface with the USB port? You could use Wired HTTP API (USB Ethernet) to take photos/shoot video and then download to the NAS.

**Reply by NickPronto** [reply]
Our current idea is to have a thumb drive on a small router, instead of a multi drive NAS. We could update to a small system running scripts for this. 

From the documentation, I don't understand how to direct the file location for the transfer, even after you pull the media list.
